apps/firelens-stability/collections/temp-golden-path-high-throughput-only/golden-path/cases/onepod-high-throughput.json

```diff
@@ -0,0 +1,9 @@
+{
+    "definitions": {
+        "include_onepod": true,
```

Comment:
I don't fully get the config yet... it feels like there's lots of duplication here between all these different files I'm reviewing...